frame/wormhole/issuing/s2s/src/lib.rs

```diff
@@ -175,7 +177,8 @@ pub mod pallet {
 				&user,
 			)?;
 
-			let backing_address = user.encode().as_slice().derive_ethereum_address();
+			let backing_address = sp_runtime::AccountId32::try_from(
```

Comment:
Why not `use sp_runtime::AccountId32` at the top?

Reply:
Just used once, so LGTM